.github/workflows/upload-test-stats.yml

```diff
@@ -44,6 +44,10 @@ jobs:
           pip3 install rockset==1.0.3
           pip3 install boto3==1.19.12
 
+      - name: Get added or modified  files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
```

Comment:
Have you been able to verify that this action returns the correct list of changed files?  It's usually straightforward if the workflow is on push or pull request, but this one is triggered when another workflow finishes.

Also I wonder if the list of changed files should be stored in `test_run_summary`.  It makes more sense to have that information stored in a table (could be a new one) keyed by SHA commit.  These tables can then be joined.

* The size `test_run_summary` is relatively big
* As Zain mentions, there might be other use cases that also require this information in the future